### ISSUE TITLE
Argument 0 for @NotNull parameter of com/intellij/openapi/roots/impl/LibraryOrderEntryImpl.setScope must not be null: Argument 0 for @NotNull parameter of com/intellij/openapi/roots/impl/LibraryOrderEntryImpl.setScope must not be null

### ISSUE THREAD

**Issue by dkincaid** [derkork/intellij-leiningen-plugin#35]
Argument 0 for @NotNull parameter of com/intellij/openapi/roots/impl/LibraryOrderEntryImpl.setScope must not be null: Argument 0 for @NotNull parameter of com/intellij/openapi/roots/impl/LibraryOrderEntryImpl.setScope must not be null

project.clj:
(defproject wordcount "0.1.0-SNAPSHOT"
  :min-lein-version "2.0.0"
  :description "FIXME: write description"
  :source-paths ["src/clj"]
  :test-paths ["test/clj" "test/java"]
  :java-source-paths ["src/java"]
  :javac-options ["-target" "1.6" "-source" "1.6" "-Xlint:-options" "-g"]
  :resource-paths ["src/resources" "test/resources"]
  :dependencies [[org.clojure/clojure "1.5.1"]
                  [org.apache.hadoop/hadoop-common "2.0.4-alpha"]])

**Comment by 256BitChris** [comment]
The problem you're having is being caused by the hadoop-annotations [2.0.4-alpha](http://bit.ly/139iaha) dependency which has an odd dependency on jdk.tools.  This didn't exist in the 2.0.3-alpha version and is probably an error in that dependency.  I've never seen the tools jar explicitly referenced like this before and it will probably cause other problems.

``` xml
<dependency>
  <groupId>jdk.tools</groupId>
  <artifactId>jdk.tools</artifactId>
  <version>1.6</version>
  <scope>system</scope>
  <systemPath>${java.home}/../lib/tools.jar</systemPath>
</dependency>
```

Either way, the plugin should handle this gracefully and I've added a fix for the exception you saw in Pull Request [36](https://github.com/derkork/intellij-leiningen-plugin/pull/36).  

I'd contact the hadoop guys and let them know about depending on the tools jar in their pom files so they can remove it.

**Comment by dkincaid** [comment]
Fantastic! Thanks for the troubleshooting and the fix.

**Comment by derkork** [comment]
Released 0.3.4 to the plugin repo. Thanks @64BitChris for providing this fix :)
